test/gui/shared/scripts/bdd_hooks.py

```diff
@@ -37,7 +37,7 @@ def hook(context):
     DEFAULT_CONFIG = {
         'localBackendUrl': 'https://localhost:9200/',
         'secureLocalBackendUrl': 'https://localhost:9200/',
-        'clientSyncTimeout': 60,
+        'clientSyncTimeout': 10,
```

Comment:
This PR seems to be dependent upon changes made in some other PRs (part 1 and 3). Does the order of merge should be Part1 -> part2 -> part3 ?

Reply:
the actual order should not affect each other. one needs to be rebased. So whichever gets merged first, it should not create affect other PR